Sources/Target/Linux/Process.cpp

```diff
@@ -327,13 +327,19 @@ ErrorCode Process::getMemoryRegionInfo(Address const &address,
     uint64_t inode;
     int nread;
 
-    if (std::fgets(buf, sizeof(buf), fp) == nullptr)
-      break;
+    if (::fgets(buf, sizeof(buf), fp) == nullptr) {
```

Comment:
Cause I find it cleaner to use `std::` for things coming from the standard C++ library, and straight up `::` for things coming from the C library. `fgets` for instance is a C function, not a C++ function.